### ISSUE TITLE
Aliases not working in Vscode terminal

### ISSUE THREAD

**Issue by BolajiAyodeji** [SarathSantoshDamaraju/lazyGit#17]
The aliases work just fine on my `git bash` but I just tried running them on my Vscode terminal and it didn't work at all
ERROR: `bash: command not found`

**Comment by SarathSantoshDamaraju** [comment]
Did you try re-starting VS code?

**Comment by BolajiAyodeji** [comment]
> Did you try re-starting VS code?

Yes, I did, still same.
Not working!

**Comment by SarathSantoshDamaraju** [comment]
It should reflect, I will look into it. But for that, I need some debug info from you, can you confirm/share the following
- [ ] Check if [this issue](https://github.com/Microsoft/vscode/issues/7263) can help you.

**If not**
- [ ]  Did you try typing the lazyGit commands from your terminal/command prompt other than `git bash`

- [ ] Share your VS code and OS details?

**Comment by BolajiAyodeji** [comment]
> It should reflect, I will look into it. But for that, I need some debug info from you, can you confirm/share the following
> 
> * [ ]  Check if [this issue](https://github.com/Microsoft/vscode/issues/7263) can help you.
> 
> **If not**
> 
> * [ ]   Did you try typing the lazyGit commands from your terminal/command prompt other than `git bash`
> * [ ]  Share your VS code and OS details?

Let me try this

**Comment by SarathSantoshDamaraju** [comment]
@BolajiAyodeji Is it fixed?

**Comment by BolajiAyodeji** [comment]
@SarathSantoshDamaraju Nope.
That issue did not help.
YES, That's what I'm trying to do, I've been using LazyGit commands on git bash, it works fine but on terminal/command it doesn't
VScode v1.30.2
Windows 10

**Comment by SarathSantoshDamaraju** [comment]
I believe issue is with the bash file. This is written for Linux systems, thus bash file is given as `.bash_profile`. If i am not wrong in windows it is `.bashrc` and this might be reason behind not able to access the commands from the Windows CMD. I need your help in it. 

> TLDR; Try pasting [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) file in `.bashrc`

1. Open the file `.bashrc` which is found in location `C:\Users\USERNAME\.bashrc`

2. If file `.bashrc` not exist then create it using below steps:

- Open Command Prompt and goto `C:\Users\USERNAME\`.
- Type command notepad `~/.bashrc`
- It generates the `.bashrc` file.
- Add [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) commands and **restart terminal**. 

Please let me know if this works.

**Comment by JoshMurph316** [comment]
adding my aliases to the .bashrc worked for me. Thank you!

**Comment by chelojimenez** [comment]
.bashrc worked for me too!

**Comment by Isaac-Svi** [comment]
@SarathSantoshDamaraju Perfect!  Fixed it for me as well.  Thanks a lot.  👍

**Comment by xmazlan** [comment]
> I believe issue is with the bash file. This is written for Linux systems, thus bash file is given as `.bash_profile`. If i am not wrong in windows it is `.bashrc` and this might be reason behind not able to access the commands from the Windows CMD. I need your help in it.
> 
> > TLDR; Try pasting [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) file in `.bashrc`
> 
> 1. Open the file `.bashrc` which is found in location `C:\Users\USERNAME\.bashrc`
> 2. If file `.bashrc` not exist then create it using below steps:
> 
> * Open Command Prompt and goto `C:\Users\USERNAME\`.
> * Type command notepad `~/.bashrc`
> * It generates the `.bashrc` file.
> * Add [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) commands and **restart terminal**.
> 
> Please let me know if this works.

perfect

**Comment by levietanh0001** [comment]
> I believe issue is with the bash file. This is written for Linux systems, thus bash file is given as `.bash_profile`. If i am not wrong in windows it is `.bashrc` and this might be reason behind not able to access the commands from the Windows CMD. I need your help in it.
> 
> > TLDR; Try pasting [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) file in `.bashrc`
> 
> 1. Open the file `.bashrc` which is found in location `C:\Users\USERNAME\.bashrc`
> 2. If file `.bashrc` not exist then create it using below steps:
> 
> * Open Command Prompt and goto `C:\Users\USERNAME\`.
> * Type command notepad `~/.bashrc`
> * It generates the `.bashrc` file.
> * Add [this](https://github.com/SarathSantoshDamaraju/lazyGit/blob/master/commands.txt) commands and **restart terminal**.
> 
> Please let me know if this works.

In my case I had to drop '&&' and input aliases like this for the commands to work:
```
alias g!='git init'
alias gs='git status'
```
